src/deps/uws.zig

```diff
@@ -1894,6 +1894,153 @@ pub const SocketAddress = struct {
     is_ipv6: bool,
 };
 
+pub const AnyResponse = union(enum) {
+    SSL: *NewApp(true).Response,
+    TCP: *NewApp(false).Response,
```

Comment:
would be good to add a .detached option here, and detach after we call end/close/abort to avoid UAF, server.zig actually just set the resp/req to null, We should detach in the same places, even if is less likely to be a Issue in static.

Reply:
There is no risk of UAF because it doesn't keep a reference to the Response pointer outside the stack